src/vorta/utils.py

```diff
@@ -240,7 +240,22 @@ def sort_sizes(size_list):
     return final_list
 
 
-def pretty_bytes(size, metric=True, sign=False, precision=1):
+def find_best_size_formatting(size_list, metric=True):
+    power, units = (
+        (10**3, ['', 'K', 'M', 'G', 'T', 'P', 'E', 'Z', 'Y'])
+        if metric
+        else (2**10, ['', 'Ki', 'Mi', 'Gi', 'Ti', 'Pi', 'Ei', 'Zi', 'Yi'])
+    )
+    min_size = min(s for s in size_list if isinstance(s, int))
+    # we tolerate a maximum precision of 1
+    n = 0
+    while abs(min_size / power**n > 0.1) and n + 1 < len(units):
+        n += 1
+    n = max(n - 1, 0)
+    return 1, units[n]
```

Comment:
```suggestion
def find_best_size_formatting(sizes, metric=True, precision=1):
    power =  10**3 if metric else 2**10
    min_size = min((s for s in sizes if isinstance(s, int)), default=None)
    if min_size is None:
        return 1, 0
    n = math.ceil(math.log(min_size, power))
    return 1, n if min_size / power**n >= 0.1**precision else n-1
```

Reply:
This simplifies the code and fixes the error.

Reply:
thanks, I'll apply it asap. so I deduce it was crashing on an empty list.. what a rookie mistake!

Reply:
>  I deduce it was crashing on an empty list.. what a rookie mistake!

Yes, error message was `src/vorta/utils.py:249: ValueError: min() arg is an empty sequence`.